src/python/pants/backend/go/lint/gofmt/subsystem.py

```diff
@@ -11,3 +14,7 @@ class GofmtSubsystem(Subsystem):
     help = "Gofmt-specific options."
 
     skip = SkipOption("fmt", "lint")
+    args = ArgsListOption(
+        example="-s -e",
+        extra_help=f"Only the following style related options are supported: {tuple(SUPPORTED_GOFMT_ARGS)}.",
```

Comment:
Maybe the following would look nicer than the tuple?

```
", ".join([f"`{x}`" for x in xs])
```

Reply:
Sure, this looks great to me!

```
  --gofmt-args="[<shell_str>, <shell_str>, ...]"
  PANTS_GOFMT_ARGS
  args
      default: []
      current value: []
      Arguments to pass directly to gofmt, e.g. `--gofmt-args='-s -e'`.
      
      Only the following style related options are supported: `-e`, `-r`, `-s`.
```